### ISSUE TITLE
asciidoctor-diagram 2.1.1

### ISSUE THREAD

**Issue by chkpnt** [asciidoctor/asciidoctorj-diagram#14]
I would like to kindly ask for a new release respecting the current version of asciidoctor-diagram. 🙏

It allows using PlantUML-features that require a preprocessing! So something like [C4-PlantUML](https://github.com/plantuml-stdlib/C4-PlantUML) can now be used to get non-ugly diagrams.

I've just tested it with a asciidoctorj-diagram 2.1.1 in my local maven repository and forcing Gradle to use it with `diagram.version("2.1.1")`: It works great :-)

**Comment by chkpnt** [comment]
Mh, now I'm confused: I've just noticed that there isn't a release 2.1.1 (rubygem) of asciidoctor-diagram. Don't know why my local test worked.

**Comment by robertpanzer** [comment]
asciidoctor-diagram 2.1.1 doesn't seem to be released yet, https://rubygems.org/gems/asciidoctor-diagram still shows 2.1.0 as latest version.
But I can release asciidoctorj-diagram 2.1.1 as soon as it is released.

**Comment by robertpanzer** [comment]
> Mh, now I'm confused: I've just noticed that there isn't a release 2.1.1 (rubygem) of asciidoctor-diagram. Don't know why my local test worked.

Interesting :D Don't hesitate to ping me again once asciidoctor-diagram 2.1.1 is released, I sometimes miss them and don't follow up immediately.

**Comment by chkpnt** [comment]
Yeah, I just updated `asciidoctorj-diagram/gradle.properties` and `gradle.properties` and executed `./gradlew asciidoctorj-diagram:publishToMavenLocal -Pskip.signing` without an error... 🤷

**Comment by chkpnt** [comment]
Okay, just did a retest. Turns out, that forcing `asciidoctorj` to use `asciidoctorj-dagram 2.1.0` instead the default one (`diagram.use()` in build.gradle) is enough to fix the issues I have with the preprocessing of PlantUML-diagrams. And I took a look into `asciidoctorj-diagram-2.1.1.jar` in my local maven repository: it contains the gem of `asciidoctor-diagram-2.1.0`. Weired that gradle didn't fail...

But I guess this issue here is not appropriate. Sorry for the confusion.
